ui/pages/home/home.component.js

```diff
@@ -332,6 +335,8 @@ export default class Home extends PureComponent {
       history.push(BUILD_QUOTE_ROUTE);
     } else if (firstPermissionsRequestId) {
       history.push(`${CONNECT_ROUTE}/${firstPermissionsRequestId}`);
+    } else if (pendingConfirmationsPrioritized.length > 0) {
```

Comment:
I've tried this branch locally and I can't see the STX status page if I do a dapp transaction on Sepolia or just a Send transaction. STX is enabled in Settings.

Reply:
I've just tested locally and encountered the same, after some debug it seems to be a remote issue since `state.metamask.smartTransactionsState?.liveness` is `false`.

Reply:
It looks like Sepolia was disabled for STX this week. Working with the team to enable it again for testing purposes. On Ethereum mainnet it works.